examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -43,3 +50,18 @@
 
 # Now let's take a look at the spatial distributions of the psd.
 epochs.plot_psd_topomap(ch_type='grad', normalize=True)
+
+# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
+f, ax = plt.subplots()
```

Comment:
That's what I always teach as well, but I'll go with whatever convention
you guys prefer...

On Mon, Jan 11, 2016 at 5:32 AM, Eric Larson <notifications@github.com>
wrote:

> In examples/time_frequency/plot_epochs_spectra.py
> <https://github.com/mne-tools/mne-python/pull/2710#discussion_r49320621>:
>
> > @@ -43,3 +50,18 @@
> >
> >  # Now let's take a look at the spatial distributions of the psd.
> >  epochs.plot_psd_topomap(ch_type='grad', normalize=True)
> > +
> > +# Alternatively, you may also create PSDs from Epochs objects with psd_XXX
> > +f, ax = plt.subplots()
>
> I think methods on figures and axes are preferred to the global plotting
> functions (e.g. plt.plot) in mpl
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mne-tools/mne-python/pull/2710/files#r49320621>.
>